docker-compose.yml

```diff
@@ -46,9 +50,14 @@ services:
       - ETCD_AUTO_COMPACTION_MODE=revision
       - ETCD_AUTO_COMPACTION_RETENTION=1000
       - ETCD_QUOTA_BACKEND_BYTES=4294967296
+    healthcheck:
+      test: ["CMD", "/opt/bitnami/scripts/etcd/healthcheck.sh"]
+      interval: 30s
```

Comment:
I prefer to decrease the interval time for `30s` to`10s`, as we can retry 3 times. Increase retry times instead of waiting time.

Reply:
Good catch. We'll add start_periods and decrease interval in the later pr.